src/GovUk.Education.ExploreEducationStatistics.Common/Services/CollectionUtils.cs

```diff
@@ -0,0 +1,12 @@
+using System.Collections.Generic;
+
+namespace GovUk.Education.ExploreEducationStatistics.Common.Services
+{
+    public static class CollectionUtils
+    {
+        public static List<T> AsList<T>(params T[] objects)
```

Comment:
The reason I did this was for brevity more than anything else, so rather than:

```
var publications = new List<Publication> {
    publication1,
    publication2
};
```

or 

```
var publications = (new Publication[] { publication1, publication2 }).ToList();
```

you can just write:

```
var publications = AsList(publication1, publication2);
```

Just thought it could help make test code a little more concise?

Reply:
That makes sense, I was mistaking it for an extension method. Thanks 🙂